wordpress-mobile/WordPress-iOS
# Issue/add icons to image aztec

This PR improves the appearance of image overlay and progress bar and add icons to it.

To test:
 - Create or edit a Post with Aztec
 - Add an image to it.
 - Check if the progress bars for upload are visible.
 - Check if the message and icon that shows in case of error are ok
 - Check if the message and icon that show when tapping on media are ok.



Needs review: @jleandroperez
